metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -944,9 +937,6 @@ def ingest_sample_data(path: Optional[str], token: Optional[str]) -> None:
         },
     }
 
-    if token is not None:
-        recipe["sink"]["config"]["token"] = token
```

Comment:
removing this means that the `--token` arg doesn't work?

Reply:
`--token` was removed, see code change 912. Token information should be retrieved from `get_url_and_token()` method which gets that information from env vars or datahubenv file